include/odp/api/spec/ipsec.h

```diff
@@ -1148,14 +1157,29 @@ typedef struct odp_ipsec_in_param_t {
  * Outbound inline IPSEC operation parameters
  */
 typedef struct odp_ipsec_out_inline_param_t {
-	/** Packet output interface for inline output operation
+	/** Packet output interface for inline outbound operation
 	 *
 	 *  Outbound inline IPSEC operation uses this packet IO interface to
 	 *  output the packet after a successful IPSEC transformation. The pktio
 	 *  must have been configured to operate in inline IPSEC mode.
 	 */
 	odp_pktio_t pktio;
 
+	/** Packet output TM queue for inline outbound operation
```

Comment:
Packets are sent out through traffic manager if the packet output mode of the pktout is ODP_PKTOUT_MODE_TM. Such pktout can be used with inline IPsec only if the platform has inline_ipsec_tm capability.

Reply:
Did you mean add this description after the title? If yes, I'll make this change in next version.